sycl/include/CL/sycl/ONEAPI/reduction.hpp

```diff
@@ -587,15 +673,29 @@ class reduction_impl : private reduction_impl_base {
   }
 
   /// Constructs a new temporary buffer to hold partial sums and returns
-  /// the accessor that that buffer.
-  template <bool IsOneWG>
-  std::enable_if_t<!IsOneWG, accessor_type>
+  /// the accessor that that buffer. Non-placeholder case.
```

Comment:
```suggestion
  /// the accessor for that buffer. Non-placeholder case.
```

Reply:
Thank you. Fixed in 0cdf646139cb6f27a28f985cd123eb96ebd540be